ci/vendor-wit.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Script to re-vendor the WIT files that Wasmtime uses as defined by a
+# particular tag in upstream repositories.
+#
+# This script is executed on CI to ensure that everything is up-to-date.
+
+set -ex
+
+dst=crates/wasi/wit/deps
+
+rm -rf $dst
+mkdir -p $dst
+
+repos="cli clocks filesystem http io random sockets"
+tag=0.2.0
```

Comment:
And maybe the same for how to update tag/WASI version?

May make sense to put all the "config" variables at the top of the script so that folks can update them without having to read the whole script and learn the implementation just to eg add support for WASI-foo to the script.